samples/micrometer-samples-boot2/src/main/resources/application-dynatrace.yml

```diff
@@ -19,3 +19,5 @@ management.metrics.export.dynatrace:
   apiToken: my-api-token
   uri: https://my-environment-id.live.dynatrace.com
   deviceId: my-custom-device
+  apiVersion: v2
```

Comment:
This should be v1 for now.

Reply:
this was wrongly committed when testing, removed the non-required lines for now.